database/migrations/2021_09_06_073358_create_courses_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCoursesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('courses', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
```

Comment:
thiếu nullable
đọc thêm link này https://stackoverflow.com/questions/36477287/why-should-i-use-not-use-a-nullable-field-laravel-mysql

Reply:
trong bảng course không có nullable do em thấy, không thể bỏ trống bất cứ field nào, nên em không set nullable, có được không a?

Reply:
trừ id, các trường timestamp, deleted_at theo mặc định thì còn lại cần nullable